deployment/hasura/migrations/Aerie/18_action_cancellation/up.sql

```diff
@@ -0,0 +1,32 @@
+alter table actions.action_run
+add column canceled boolean not null default false;
+
+comment on column actions.action_run.canceled is e''
+  'Whether the user has requested that this action be cancelled.';
+
+create function actions.notify_action_run_cancel_requested()
+  returns trigger
+  security definer
+  language plpgsql as $$
+begin
+  -- Notify if canceled is set to true
+  if NEW.canceled then
+    perform pg_notify('action_run_cancel_requested', json_build_object(
+        'action_run_id', NEW.id,
+        'canceled', NEW.canceled
+    )::text);
+  end if;
+
+  return null;
+end$$;
+
+create trigger notify_action_run_cancel_requested
+  after update on actions.action_run
+  for each row
+  when (
+    NEW.canceled = true
```

Comment:
Nonblocking: SQL supports standard `if(boolean)`, you don't need to do `if(boolean = true)` :-P